### PR TITLE
Fix File Permission Errors on Windows

### LINKED ISSUE #4: Unable to build frc-docs: permission error
Since sphinxext-toptranslators was added to frc-docs, I'm unable to build frc-docs. 

```
PS C:\Users\Joe\Documents\Robotics\frc-docs> ./make.bat html
Running Sphinx v3.1.1
c:\users\joe\appdata\local\programs\python\python37-32\lib\site-packages\sphinx_tabs\tabs.py:341: RemovedInSphinx40Warning: The app.add_javascript() is deprecated. Please use app.add_js_file() instead.
  app.add_javascript(path)
c:\users\joe\appdata\local\programs\python\python37-32\lib\site-packages\sphinx\deprecation.py:52: RemovedInSphinx40Warning: sphinx.builders.html.DirectoryHTMLBuilder is deprecated. Check CHANGES for Sphinx API modifications.
  return getattr(self._module, name)
hoverxref extension is not fully configured. Tooltips may not work as expected. Check out the documentation for hoverxref_project and hoverxref_version configuration options.
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 342 source files that are out of date
updating environment: [new config] 342 added, 0 changed, 0 removed
reading sources... [  2%] docs/contributing/frc-docs/top-translators
Exception occurred:
  File "c:\users\joe\appdata\local\programs\python\python37-32\lib\shutil.py", line 398, in _rmtree_unsafe
    os.unlink(fullname)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\Joe\\AppData\\Local\\Temp\\tmp4jzst04z\\.git\\objects\\pack\\pack-166749dcf6d7c9bdbeed3c042a811ab2f309cda4.idx'
The full traceback has been saved in C:\Users\Joe\AppData\Local\Temp\sphinx-err-xdfkquf6.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

It says access is denied, however, I'm able to delete that file and directory tree manually.

The log is attached
[sphinx-err-xdfkquf6.log](https://github.com/wpilibsuite/sphinxext-toptranslators/files/4915833/sphinx-err-xdfkquf6.log)

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -1,29 +1,23 @@
 name: Test and Deploy
-on:
-  pull_request:
-    branches: 
-      - master
-  push:
-    branches:
-      - master
-  create:
```
Comment (Daltz333) [inline .github/workflows/workflow.yml:9]: Now it won't run on tagging, breaks logic.

**Review by Daltz333 — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (TheTripleV) [reply .github/workflows/workflow.yml:9]: A tag counts as a push iirc, no?

Comment (TheTripleV) [reply .github/workflows/workflow.yml:9]: I'll add tags back in tho

File: sphinxext/toptranslators.py [hunk 2]
```diff
@@ -1,22 +1,49 @@
...
+    os.remove(name)
+
+
+# Delete directory if exists
+def del_directory_exists(directory: str):
+    dirpath = Path(directory)
+    if dirpath.exists() and dirpath.is_dir():
+        shutil.rmtree(dirpath, onerror=del_rw)
+    else:
+        print("Directory does not exist!")
```
Comment (Daltz333) [inline sphinxext/toptranslators.py:35]: No need for the print anymore. Just debug

**Comment by sciencewhiz** [comment]
This fixes #4 for me.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**